app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,40 @@
+/* universal */
+
+body {
+padding-top: 60px;
```

Comment:
please view this [pull request](https://github.com/lrmi0808/sample_app/pull/2/files ) and fix the ident like it